airflow/providers/celery/provider.yaml

```diff
@@ -60,3 +60,256 @@ sensors:
 executors:
   - airflow.providers.celery.executors.celery_executor.CeleryExecutor
   - airflow.providers.celery.executors.celery_kubernetes_executor.CeleryKubernetesExecutor
+
+config:
+  celery_kubernetes_executor:
+    description: |
+      This section only applies if you are using the ``CeleryKubernetesExecutor`` in
+      ``[core]`` section above
```

Comment:
Is "section above" still correct? It is now in a different config right? Same comment for all occurrences of "section above"

Reply:
Ah. Good spot :). It makes little sense anyway :).

Reply:
I also corrected all those missing ".", some inconsistencies. and reformatted the descriptions slightly. For now only in celary part - but later will do it as well for others that I will extract  CC: @ferruzzi.